strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/configuration/StoragesConfigurationControllerTestIT.java

```diff
@@ -250,6 +264,11 @@ public void testAddGetRepository()
                      "localhost",
                      repository1.getProxyConfiguration().getHost());
 
+        PoolStats poolStatsRepository2 = proxyRepositoryConnectionPoolConfigurationService.getPoolStats(secondRepositoryUrl);
+
+        assertEquals("Max connections for proxy repository not set accordingly!",
+                maxConnectionsRepository2.intValue(),
```

Comment:
Cosmetic: Could you please follow the coding convention's indentation? Thanks!

Reply:
Sorry, added your code style settings file to my IDE now.